src/debian/pacemaker_override.conf.j2

```diff
@@ -1,7 +1,6 @@
 [Unit]
-After=rbdmap.service
-After=libvirt-guests.service
-After=ceph.target
```

Comment:
We do not need to want for ceph and rbdmap?

Reply:
libvirt-guests.service is disabled right now.
ceph is already a dependency of rbdmap.service
as for rbdmap, it already is a dependency of libvirtd (via remote-fs.target), so I can replace it with a direct dependency of pacemaker to libvirtd.